### ISSUE TITLE
you can mount in wsg flag room?

### ISSUE THREAD

**Issue by Ethugx** [SunwellWoW/Sunwell-TBC-Bugtracker#3875]
Description: 

How it works: 

How it should work: 

Source (you should point out proofs of your report, please give us some source):

**Comment by username-is-not-taken** [comment]
Yes, and some other areas in BGs where mounting is possible and should not be:
- The ground floor of the horde AV towers (not sure if you should be able to mount on the wooden parts of the towers, but the concrete floors should dismount the player).
- The outer ring of the alliance AV bunkers' top floor (the inner area, where the second stairs end and the flag stands, should be mountable as far as I know, but the outer area where the archers stand should dismount the player).

If someone can confirm these areas, as they're only taken from my memory, that'd be great. Google gave me nothing.
